django-backend/src/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django==2.2.3

```

Comment:
Shouldn't we put shelve here too?

Reply:
It is a part of the Python standard library.